starknet_devnet/postman_wrapper.py

```diff
@@ -170,3 +177,82 @@ def load_mock_messaging_contract_in_l1(self, starknet, contract_address):
         self.l1_to_l2_message_filter = self.mock_starknet_messaging_contract.w3_contract.events.LogMessageToL2.createFilter(
             fromBlock="latest"
         )
+
+
+class Postman:
+    """
+    Postman class copied from starknet code base.
```

Comment:
The docstring can be expanded with the link to the exact line/file (from the exact version - https://github.com/starkware-libs/cairo-lang/tree/v0.10.1/src)

Reply:
Ok, will do.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/319/commits/58ad6199af01c3c49bc33ab144bad9728e167a80